### PR TITLE
Adds "Managing Releases" to help center docs

### PR DESCRIPTION
**Pull request by nihalathanikkal**

### CODE REVIEW

**Review by peicodes — APPROVED** [review]
You'll need to update `docs.yml` for it to show up on the nav

File: fern/docs/content/help/deployments/managing-releases.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+For example, you can assign Custom Release Tags to point to a version of your prompt in both your “Production” and “Staging” environments.
+
+This method offers you the ability to manage your releases according to your own specific workflow and versioning requirements.
+
+<Callout intent="info">
+Note that Custom Release Tags follow a specific format: less than or equal to 150 characters, begin and end with a Regex word character (i.e., letter, number, or underscore), contain only word characters, dashes or dots, and do not contain more than one dot in a row
+</Callout>
+
+### 2. Use Static Release Tags
```
Comment (dvargas92495) [inline fern/docs/content/help/deployments/managing-releases.mdx:36]: [nit] 3.

File: fern/docs/content/help/deployments/managing-releases.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+1. Within the Update Existing tab of the Deploy Prompt modal in a Prompt Sandbox
+![Updating Release Tags from Deploy Modal](https://storage.googleapis.com/vellum-public/help-docs/release-tags-update-on-deploy.png)
+
+2. By opening the Assign Release Tags modal from the Releases tab of the Deployment Details page
+![Creating Release Tags Post Deployment](https://storage.googleapis.com/vellum-public/help-docs/release-tags-post-deploy.png)
+
+### Pinning to a Deployment Release Tag in Code
+
+Both the Vellum `execute_prompt` and `execute_workflow` API’s accept an optional `release_tag` parameter to pin your request to a specific release tag. When these release tags are updated via the Vellum Playground to point to new versions of a prompt or workflow, these changes will reflect in these requests.
```
Comment (dvargas92495) [inline fern/docs/content/help/deployments/managing-releases.mdx:68]: We introduce `execute_workflow` here, but don't mention workflows above. We should clarify that the release tag strategies are available for both wherever relevant above

File: fern/docs/content/help/deployments/managing-releases.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+--data '
+{
+  "inputs": [
+    {
+      "type": "STRING",
+      "name": "string",
+      "value": "string"
+    }
+  ],
+  “release_tag”: “my-awesome-tag”
```
Comment (dvargas92495) [inline fern/docs/content/help/deployments/managing-releases.mdx:90]: [nit] Would prefer using either `production`/`staging`/`development`/`1.x.x` to ground with a use case

File: fern/docs/content/help/deployments/managing-releases.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+If no `release_tag` parameter is provided, the request will default to the latest version of that Prompt Deployment.
+
+More details on how to do this: [Vellum API Docs](https://docs.vellum.ai/api-reference/api-reference/execute-prompt)
+
+<CodeBlock title="Example Request">
+```
+curl -X POST \
+--url "https://predict.vellum.ai/v1/execute-prompt" \
+--header "Content-Type: application/json" \
+--header "X_API_KEY: <apiKey>" \
```
Comment (dvargas92495) [inline fern/docs/content/help/deployments/managing-releases.mdx:79]: [nit] I like to encourage environment variable usage in code snippets. In bash, it's `"X_API_KEY: $VELLUM_API_KEY"`

**Review by dvargas92495 — APPROVED** [review]

**Pull request merged**